runtime/permissions/lib.rs

```diff
@@ -3438,7 +3441,8 @@ mod tests {
   fn test_request() {
     set_prompter(Box::new(TestPrompter));
     let parser = TestPermissionDescriptorParser;
-    let mut perms: Permissions = Permissions::none_without_prompt();
```

Comment:
I've changed this to `none_with_prompt()` because it seemed like a mistake, since later in the test we manipulate the prompt value. I feel like this is why it has been overlooked in the first place, though I could be wrong